web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -324,119 +314,182 @@ Migrating your existing app to the new auth system is a two-step process:
 
 :::info Migrating a deployed app
 
-While going through these steps, we will focus first on doing the changes locally and your local development database.
+While going through these steps, we will focus first on doing the changes locally (including your local development database).
 
-Once we confirm everything works well locally, we will apply the same changes to the deployed app.
+Once we confirm everything works well locally, we will apply the same changes to the deployed app (including your production database).
 
 **We'll put extra info for migrating a deployed app in a box like this one.**
 :::
 
 #### 1. Migrate to the New Auth System
 
-You can follow these steps to migrate to the new auth system:
-
-1. Upgrade Wasp to the latest 0.12.X version (as described [above](https://discord.gg/rzdnErX)).
-
-
-1. Bump the version to `^0.12.0` in `main.wasp` (if you haven't already done this when [migrating the app to the new structure](#migrating-your-project-to-the-new-structure)).
```

Comment:
We expect them to read this doc in sequence, so I removed this and just mentioned above that we expect them to have done this already, this will cover the case well enough if they are jumping through the docs.